azurelinuxagent/common/rdma.py

```diff
@@ -82,11 +83,16 @@ def is_driver_loaded(self):
         """Check if the network module is loaded in kernel space"""
         cmd = 'lsmod | grep %s' % self.driver_module_name
         status, loaded_modules = shellutil.run_get_output(cmd)
+        logger.info('RDMA: Checking if the module loaded.')
         if loaded_modules:
+            logger.info('RDMA: module loaded.')
             return True
+        logger.info('RDMA: module not loaded.')
 
     def reboot_system(self):
         """Reboot the system. This is required as the kernel module for
            the rdma driver cannot be unloaded with rmmod"""
-        logger.info('System reboot')
-        shellutil.run('shutdown -r now')
+        logger.info('Rebooting system.')
```

Comment:
Should probably add the "RDMA:" prefix to this message as well so we know the reboot was initiated from the RDMA code